config/dms.php

```diff
@@ -52,7 +54,7 @@
     |
     | @var boolean
     */
-    'are_guest_public_search_enabled' => getenv('DMS_ARE_GUEST_PUBLIC_SEARCH_ENABLED') ?: true,
+    'are_guest_public_search_enabled' => env('KBOX_ENABLE_GUEST_NETWORK_SEARCH', env('DMS_ARE_GUEST_PUBLIC_SEARCH_ENABLED', true)),
```

Comment:
Will we remember to un-wrap the redundant variables in a couple of versions from now? What's your view on how long to support this?

Reply:
I guess we will remember, my idea is to have two versions before removing the support, so we will be sure we don't need to test anything below 0.22